src/structures/slashCommands/gamble.ts

```diff
@@ -8,6 +8,11 @@ const gamble: RESTPostAPIApplicationCommandsJSONBody = {
 		description: "How many coins are you willing to lose?",
 		type: ApplicationCommandOptionType.INTEGER,
 		required: true,
+	},
+	{
+		name: "multiplier",
+		description: "How much do you want to multiply your bet by?",
```

Comment:
Maybe specify the default here?

Reply:
In the description?

Reply:
Yes, I don't think Discord provides another way for this yet.

Reply:
Alright, had to make sure since I am not caught up with slash commands.